src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```diff
@@ -16,18 +16,10 @@
     <DevelopmentDependency>true</DevelopmentDependency>
     <Description>Coverlet is a cross platform code coverage library for .NET Core, with support for line, branch and method coverage.</Description>
     <PackageTags>coverage testing unit-test lcov opencover quality</PackageTags>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-
-    <!-- Items in the Content group are placed in this folder in the NuGet package. -->
-    <ContentTargetFolders>build</ContentTargetFolders>
-
-    <!-- Build tasks should not be added to the lib folder. -->
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-    <IncludeSymbols>true</IncludeSymbols>
-    <IncludeSources>true</IncludeSources>
-
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackBuildOutputs</TargetsForTfmSpecificContentInPackage>
+    <ContentTargetFolders>build\$(TargetFramework)\</ContentTargetFolders>
```

Comment:
❗️ This change broke coverlet support for projects targeting .NET Framework prior to 4.6.1. Placing the props/targets in a framework-specific folder makes them dependent on the declared target framework of the project, which we do not want. We only want them to be dependent on the target framework of the currently executing version of MSBuild.